modules/atomic-widgets/elements/div-block.php

```diff
@@ -105,6 +105,10 @@ protected function add_render_attributes() {
 				$settings['classes'] ?? '',
 			],
 		] );
+
+		if ( isset( $settings['aria_hidden'] ) && $settings['aria_hidden'] ) {
+			$this->add_render_attribute( '_wrapper', 'aria-hidden', 'true' );
+		}
```

Comment:
Are there cases in which the div block won't have an `aria-hiddent` attr?
Where do we set the `aria-hidden`?

Reply:
Do we know what purpose `aria-hidden` has in this context? Is it needed, is it an SEO/accessibility issue?

Reply:
@ronenelementor As far as I understand,  the purpose of aria hidden is to prevent from using an element that is not visible.
In our case there is a focused element that is inside an element that contains an `aria-hidden` attr set to true.
Which makes no sense.